HelloWorldMetal/README.md

```diff
@@ -0,0 +1,45 @@
+### Requirements
+
+- XCode 11.0 or above
+- iOS 12.0 or above
+
+## The HelloWorldMetal Example
+
+HelloWorldMetal is a simple image classification application that demonstrates how to use PyTorch C++ libraries with Metal support. The code is written in Swift and uses Objective-C as a bridge.
+
+We’ll be using the mobilenetv2 model as an example. Since the mobile GPU features are currently in the prototype stage, you’ll need to build a custom pytorch binary from source. For the time being, only a limited number of operators are supported, and certain client side APIs are subject to change in the future versions.
+
+### Model Preparation
```

Comment:
I think we can just add a link to the OSS Metal tutorial.

Reply:
Sonds good. A link to `(Prototype) Use iOS GPU in PyTorch` is added.